locales/en.yml

```diff
@@ -296,6 +296,7 @@ en:
     show_shortcuts: "Show Shortcuts"
     singleview: "Toggle single comment edit view"
     thismenu: "Open this menu"
+    jumptoqueue: "Jump to specific queue"
```

Comment:
Use `jump_to_queue` to match the convention. (I know not all keys follow the convention yet..)